thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/AnomalyFunctionBean.java

```diff
@@ -34,6 +34,8 @@
 
   private boolean isActive = true;
 
+  private boolean metricSum = false;
```

Comment:
Which metric is going to be used for calculating the metric sum? Please make this part generic. Specifically, let users specify which metrics should be retrieved for calculating metric sum.